addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -17,7 +17,7 @@
                             <field name="company_id" invisible="1"/>
                             <field name="partner_id" invisible="1"/>
                             <field name="sale_order_id" invisible="1"/>
-                            <field name="employee_id" widget="many2one_avatar_user"/>
+                            <field name="employee_id" widget="many2one_avatar_user" context="{'create_project_employee_mapping': company_id}"/>
```

Comment:
Comment for me

To check if we can do in another way to avoid context key.

Reply:
We cannot really do the same in another way and so we will let this context key. 🙂